### PR TITLE
applications: nrf5340_audio: Added capabilities doc for the controller

### PR DESCRIPTION
**Pull request by erikrobstad**
Based on CIS/BIS BAP tables.

### CODE REVIEW

**Comment by erikrobstad** [comment]
Hmmm, compliance fails with "No binary files allowed.". I checked now and it does not look like there are other PDF files in the nrf folder.
Is this a blocker for using .pdf @greg-fer ?

**Review by greg-fer — CHANGES_REQUESTED** [review]
PDFs can't go into the main repo. I'll upload it to the FTP.

**Comment by koffes** [comment]
@greg-fer: Should we then link to it? Can we have it as a markdown?

**Review by greg-fer — APPROVED** [review]

File: doc/nrf/libraries/bin/bt_ll_acs_nrf53/index.rst [hunk 1]
```diff
@@ -13,13 +13,18 @@ This controller is compatible with the HCI RPMsg driver provided by the |NCS| Bl
...
+   The LE Audio controller for nRF5340 associated with this version of the |NCS| comes with the QDID #181316.
 
 Configuration
 *************
 
 The controller can be enabled with the :Kconfig:option:`CONFIG_BT_LL_ACS_NRF53` set to ``y``.
 
+The number of Connected Isochronous Streams (CIS) you can use in a Connected Isochronous Group (CIG) and the number of Broadcast Isochronous Streams (BIS) you can use in a Broadcast Isochronous Group (BIG)
+are dependent on which configurations you use.
+These are configurations like sample frequency, bitrate and number of retransmissions.
```
Comment (alexsven) [inline doc/nrf/libraries/bin/bt_ll_acs_nrf53/index.rst:25]:
```suggestion
The relevant configurations are bitrate and number of retransmissions.
```
Sample frequency does not affect this, the packet on air can be 120 bytes regardless if it is sampled in 48kHz or 8kHz

**Review by alexsven — CHANGES_REQUESTED** [review]

**Review by alexsven — APPROVED** [review]

File: doc/nrf/libraries/bin/bt_ll_acs_nrf53/index.rst [hunk 2]
```diff
@@ -13,13 +13,18 @@ This controller is compatible with the HCI RPMsg driver provided by the |NCS| Bl
...
-   The LE Audio controller for nRF5340 associated with this version of the |NCS| comes with the QDID 181316.
+   The LE Audio controller for nRF5340 associated with this version of the |NCS| comes with the QDID #181316.
 
 Configuration
 *************
 
 The controller can be enabled with the :Kconfig:option:`CONFIG_BT_LL_ACS_NRF53` set to ``y``.
 
+The number of Connected Isochronous Streams (CIS) you can use in a Connected Isochronous Group (CIG) and the number of Broadcast Isochronous Streams (BIS) you can use in a Broadcast Isochronous Group (BIG)
+are dependent on which configurations you use.
```
Comment (FrancescoSer) [inline doc/nrf/libraries/bin/bt_ll_acs_nrf53/index.rst:24]:
```suggestion
The number of Connected Isochronous Streams (CIS) you can use in a Connected Isochronous Group (CIG) and the number of Broadcast Isochronous Streams (BIS) you can use in a Broadcast Isochronous Group (BIG) are dependent on which configurations you use.
```

File: doc/nrf/libraries/bin/bt_ll_acs_nrf53/index.rst [hunk 3]
```diff
@@ -13,13 +13,18 @@ This controller is compatible with the HCI RPMsg driver provided by the |NCS| Bl
...
+The most relevant configurations are bitrate and number of retransmissions.
+For the matrix of supported CIS in a CIG and BIS in a BIG, see the `Capabilities for the LE Audio Controller Subsystem for nRF5340`_ document, which is based on the Basic Audio Profile (BAP) tables.
```
Comment (FrancescoSer) [inline doc/nrf/libraries/bin/bt_ll_acs_nrf53/index.rst:26]:
```suggestion
See the `Capabilities for the LE Audio Controller Subsystem for nRF5340`_ document for a matrix table listing the supported CIS in a CIG and the supported BIS in a BIG.
The document is based on the Basic Audio Profile (BAP) tables.
```

File: doc/nrf/libraries/bin/bt_ll_acs_nrf53/index.rst [hunk 4]
```diff
@@ -13,13 +13,18 @@ This controller is compatible with the HCI RPMsg driver provided by the |NCS| Bl
 When the communication between the application and the network core on a device is handled by the HCI RPMsg driver, this link layer controller can be used to handle time critical low level communication and the radio.
 
 .. note::
-   The LE Audio controller for nRF5340 associated with this version of the |NCS| comes with the QDID 181316.
+   The LE Audio controller for nRF5340 associated with this version of the |NCS| comes with the QDID #181316.
```
Comment (FrancescoSer) [inline doc/nrf/libraries/bin/bt_ll_acs_nrf53/index.rst:16]:
```suggestion
   The LE Audio controller for nRF5340 associated with this version of the |NCS| comes with the QDID ``#181316``.
```

**Review by FrancescoSer — CHANGES_REQUESTED** [review]

File: doc/nrf/libraries/bin/bt_ll_acs_nrf53/index.rst (hunk 2 above)
Comment (erikrobstad) [reply doc/nrf/libraries/bin/bt_ll_acs_nrf53/index.rst:24]: Done

File: doc/nrf/libraries/bin/bt_ll_acs_nrf53/index.rst (hunk 4 above)
Comment (erikrobstad) [reply doc/nrf/libraries/bin/bt_ll_acs_nrf53/index.rst:16]: Done

File: doc/nrf/libraries/bin/bt_ll_acs_nrf53/index.rst (hunk 3 above)
Comment (erikrobstad) [reply doc/nrf/libraries/bin/bt_ll_acs_nrf53/index.rst]: Done, with some modifications

**Review by FrancescoSer — APPROVED** [review]

**Review by koffes — APPROVED** [review]

**Pull request merged**